Dockerfile

```diff
@@ -4,13 +4,18 @@ RUN apt-get update && apt-get install -y gfortran liblapack-dev libblas-dev
 RUN wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh && \
   bash miniconda.sh -b -p $HOME/miniconda
 
-COPY ./requirements.txt requirements.txt
+COPY ./deep_qa /deep_qa
+COPY ./deep_qa/requirements.txt requirements.txt
 RUN pip3 install -r requirements.txt
-RUN pip3 install https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.11.0rc0-cp35-cp35m-linux_x86_64.whl
 
 COPY ./src /src
 
 ENV PATH $HOME/miniconda/bin:$PATH
 ENV PYTHONHASHSEED 2157
+ENV PYTHONPATH /deep_qa:$PYTHONPATH
 
-ENTRYPOINT ["python", "src/main/python/server.py"]
+EXPOSE 50051
+
+COPY ./eb4b1b35/ /efs/data/dlfa/models/eb4b1b35/
```

Comment:
It is a hash, and I'm not intending for this to be a final solution.  The hash is a stable identifier for a particular set of parameters, and it's a way to find the saved model given the parameters.  I'm assuming that a final solution to the things we've been discussing will involve some better way to locate the model files given the parameters, and so this line will be changed accordingly.  The hash you want to use depends on the model parameters, for example, so hard-coding a particular hash in here won't work.